parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -308,14 +456,16 @@ pub mod pallet {
 		id: H256,
 		origin: ParaId,
 		message: BoundedVec<u8, MaxMessageSize>,
+		command: Command,
+		location: MultiLocation,
```

Comment:
It shouldn't be necessary to add this location field to the Message type, with the design I suggest in a comment below.

Reply:
https://github.com/Snowfork/snowbridge/pull/940#discussion_r1333152954

https://github.com/Snowfork/snowbridge/pull/940#discussion_r1325343050

https://github.com/Snowfork/snowbridge/pull/940#discussion_r1333114054